### PR TITLE
Feature/landing

### PR DESCRIPTION
**Pull request by Khibar**

### CODE REVIEW

File: src/components/layouts/components/DefaultLayoutFooter.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  name: 'DefaultLayoutFooter'
+}
+</script>
+
+<style lang="scss" scoped>
+.footer-wrapper--flex {
+  display: flex;
+  flex-direction: column;
+}
+.footer-wrapper{
```
Comment (IlyaDonskikh) [inline src/components/layouts/components/DefaultLayoutFooter.vue]: space before `{`. Here and in other lines.

Comment (IlyaDonskikh) [inline src/components/layouts/components/DefaultLayoutFooter.vue]: Add an extra line between the end of the element and the beginning of a new one.

File: src/components/layouts/components/DefaultLayoutFooter.vue [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      &:visited {
+        color: #ffffff;
+      }
+    }
+  }
+
+  .footer-content{
+    margin: 0 auto;
+
+    &__companyName{
```
Comment (IlyaDonskikh) [inline src/components/layouts/components/DefaultLayoutFooter.vue]: Do not use camel case for the naming. `company-name`.

File: src/components/layouts/default.vue [hunk 3]
```diff
@@ -1,97 +1,30 @@
...
     </el-main>
+    <el-footer class="default-layout__footer">
+      <DefaultLayoutFooter />
+    </el-footer>
   </el-container>
 </template>
 
 <script>
 import { mapGetters, mapActions } from 'vuex'
+import DefaultLayoutNavbar from './components/DefaultLayoutNavbar.vue'
```
Comment (IlyaDonskikh) [inline src/components/layouts/default.vue]: I never ask to rebuild the header. Only fix the background color. Rollback.

File: src/components/pages/home/components/HomeBenefits.vue [hunk 4]
```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
```
Comment (IlyaDonskikh) [inline src/components/pages/home/components/HomeBenefits.vue]: Name of the main css-class follow the name of JS-class. `.home-benefits`.

The name of the other classes according to the BEM-documentation.

File: src/components/pages/home/components/HomeCTA.vue [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    &:hover{
+      border: solid 2px #47b475;
+      background-color: #ffffff;
+      color: #47b475;
+    }
+  }
+}
+
+@media (max-width: 912px) {
+
```
Comment (IlyaDonskikh) [inline src/components/pages/home/components/HomeCTA.vue]: Need to delete extra line.

File: src/components/layouts/components/DefaultLayoutFooter.vue [hunk 6]
```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="footer-wrapper footer-wrapper--flex">
```
Comment (IlyaDonskikh) [inline src/components/layouts/components/DefaultLayoutFooter.vue]: Wrong modifier naming.

File: src/components/layouts/components/DefaultLayoutFooter.vue (hunk 1 above)
Comment (Khibar) [reply src/components/layouts/components/DefaultLayoutFooter.vue]: fixed the space on line 45: 40c926930f36163ff480ab39095d8c1b996b2fdf

Comment (Khibar) [reply src/components/layouts/components/DefaultLayoutFooter.vue]: style updated: d6936732de86ddfe8feb57058a6eae8029fce066

File: src/components/layouts/components/DefaultLayoutFooter.vue (hunk 2 above)
Comment (Khibar) [reply src/components/layouts/components/DefaultLayoutFooter.vue]: name changed: style updated: d6936732de86ddfe8feb57058a6eae8029fce066

File: src/components/layouts/default.vue (hunk 3 above)
Comment (Khibar) [reply src/components/layouts/default.vue]: header rolled back: d6936732de86ddfe8feb57058a6eae8029fce066

File: src/components/pages/home/components/HomeBenefits.vue (hunk 4 above)
Comment (Khibar) [reply src/components/pages/home/components/HomeBenefits.vue]: naming fixed: d6936732de86ddfe8feb57058a6eae8029fce066

File: src/components/pages/home/components/HomeCTA.vue (hunk 5 above)
Comment (Khibar) [reply src/components/pages/home/components/HomeCTA.vue]: style updated: d6936732de86ddfe8feb57058a6eae8029fce066

File: src/components/layouts/components/DefaultLayoutFooter.vue (hunk 6 above)
Comment (Khibar) [reply src/components/layouts/components/DefaultLayoutFooter.vue]: modified: d6936732de86ddfe8feb57058a6eae8029fce066

File: src/components/layouts/default.vue [hunk 7]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <el-container>
-    <el-header>
+  <el-container class="default-layout">
```
Comment (IlyaDonskikh) [inline src/components/layouts/default.vue]: How do you build the class name? The path is layouts/default. Means the class name is `layouts-default`.

File: src/components/pages/home/Index.vue [hunk 8]
```diff
@@ -1,12 +1,38 @@
...
+    HomeHow,
+    HomeFAQ
+  },
   metaInfo () {
     return { title: this.t('title') }
   }
 }
 </script>
+
+<style lang="scss" scoped>
```
Comment (IlyaDonskikh) [inline src/components/pages/home/Index.vue]: For what if you do not use it?

File: src/components/pages/home/components/HomeBenefits.vue [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+    <el-row
+      class="home-benefits"
+      type="flex"
+      justify="center"
+    >
+      <el-col
+        :xs="24"
+        :sm="24"
+        :md="6"
+        class="home-benefit-col"
```
Comment (IlyaDonskikh) [inline src/components/pages/home/components/HomeBenefits.vue]: How do you name the classes in that file? What is the logic behind this?

File: src/components/pages/home/components/HomeCTA.vue [hunk 10]
```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="home-cta">
+    <div class="cta-content">
```
Comment (IlyaDonskikh) [inline src/components/pages/home/components/HomeCTA.vue]: How did you get `cta-content` in BEM? `home-cta` -> `home-cta-content` and etc.

File: src/components/pages/home/components/HomeDescription.vue [hunk 11]
```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <el-row class="home-description">
+      <el-col
+        :sm="24"
+        :md="12"
+      >
+        <div class="description-img-box" />
```
Comment (IlyaDonskikh) [inline src/components/pages/home/components/HomeDescription.vue]: All files has same naming mistakes.

File: src/components/layouts/default.vue (hunk 7 above)
Comment (Khibar) [reply src/components/layouts/default.vue]: changed - 0649375b9070c129af1e88f0e3aac7b02db80848

File: src/components/pages/home/Index.vue (hunk 8 above)
Comment (Khibar) [reply src/components/pages/home/Index.vue]: removed - 0649375b9070c129af1e88f0e3aac7b02db80848

File: src/components/pages/home/components/HomeBenefits.vue (hunk 9 above)
Comment (Khibar) [reply src/components/pages/home/components/HomeBenefits.vue]: updated - 0649375b9070c129af1e88f0e3aac7b02db80848

File: src/components/pages/home/components/HomeCTA.vue (hunk 10 above)
Comment (Khibar) [reply src/components/pages/home/components/HomeCTA.vue]: updated - 0649375b9070c129af1e88f0e3aac7b02db80848

File: src/components/pages/home/components/HomeDescription.vue (hunk 11 above)
Comment (Khibar) [reply src/components/pages/home/components/HomeDescription.vue]: updated - 0649375b9070c129af1e88f0e3aac7b02db80848

**Pull request merged**